src/Pages/AboutUs/AboutUs.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import './AboutUs.css';
+import AboutUsCard from './AboutUsCard.js';
+import AboutUsCircle from './AboutUsCircle.js';
+import {getAllOfficers} from '../../APIFunctions/OfficerManager';
+import Header from '../../Components/Header/Header';
+import Footer from '../../Components/Footer/Footer';
+class AboutUs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      officers: [],
+      headerProps: {
+        title: 'About Us'
+      }
+    };
+  }
+
+  componentDidMount(){
+    getAllOfficers().then(response => {
+      this.setState({
+        officers: response.responseData
+      });
+    });
+  }
+
+  render() {
+    return (
+      <>
+        <Header {...this.state.headerProps} />
+        <main className="officer-body">
+          <section className="exec-container">
+            <h1>Executive Team</h1>
+            <div className="grid-container">
+              {
+                this.state.officers.map(info => {
```

Comment:
Placeholder text, like in `aboutUsManager.js` would be nice. Something like "There's no one here..." or whatever, but it's not important.